setuptools/tests/test_egg_info.py

```diff
@@ -454,9 +454,7 @@ def test_extras_require_with_invalid_marker_in_req(self, tmpdir_cwd, env):
     def test_provides_extra(self, tmpdir_cwd, env):
         self._setup_script_with_requires(
             'extras_require={"foobar": ["barbazquux"]},')
-        environ = os.environ.copy().update(
-            HOME=env.paths['home'],
-        )
+
```

Comment:
Don't you need to remove the references in calls to `environment.run_setup_py(..., env=environ)` too?

Reply:
Indeed.